src/components/MessageLogger/MessageList.module.scss

```diff
@@ -1,17 +1,28 @@
 @use "@styles/overrideDefaultMethods.scss";
 @use "@styles/styles.scss";
 
-#containerMessages {
+#faultListWrapper {
   background-color: styles.$default-background;
   width: 100%;
-  height: 100%;
+  height: 90%;
   overflow-x: hidden;
   overflow-y: auto;
+  background-color: #fec8c8;
+  border-radius: 2%;
+}
+
+#warningListWrapper {
+  background-color: styles.$default-background;
+  width: 100%;
+  height: 90%;
+  overflow-x: hidden;
+  overflow-y: auto;
+  background-color: #f3e4ad;
 }
 
 .lineMsgUl {
   width: 100%;
-  padding-top: 0.3rem;
+  padding-bottom: 0.3rem;
   @include overrideDefaultMethods.override-ul;
```

Comment:
override styles should be manually applied, they should be applied by default to the whole document. You can probably include the overrideDefaults.scss to the index.html or main or app.